draft-ietf-quic-datagram.md

```diff
@@ -242,9 +242,12 @@ priorities to DATAGRAM frames with respect to each other and to QUIC streams.
 
 Although DATAGRAM frames are not retransmitted upon loss detection, they are
 ack-eliciting ({{!RFC9002}}). Receivers SHOULD support delaying
-ACK frames (within the limits specified by max_ack_delay) in reponse to receiving
-packets that only contain DATAGRAM frames, since the timing of these
-acknowledgements is not used for loss recovery.
+ACK frames (within the limits specified by max_ack_delay) in response to
+receiving packets that only contain DATAGRAM frames, since the sender takes no
+action if these packets are temporarily unacknowledged. Receivers SHOULD
+continue to send ACK frames when conditions indicate a packet might be lost,
+since the packet's payload is unknown to the receiver, and when dictated by
```

Comment:
What does the "payload is unknown" mean?

Reply:
Unknown as in "you do not know the contents of a packet that you never received", so if you see packet numbers 1,2,3,5,6,7,8 you don't know if packet 4 was datagram-only or not, so you shouldn't treat 4 as datagram-only

Reply:
Ah, got it.